types/google-apps-script/google-apps-script.card-service.d.ts

```diff
@@ -686,16 +751,79 @@ declare namespace GoogleAppsScript {
       addUpdateContent(content: string, contentType: ContentType): UpdateDraftBodyAction;
       setUpdateType(updateType: UpdateDraftBodyType): UpdateDraftBodyAction;
     }
+    /**
+     * The fixed footer shown at the bottom of an add-on Card.
+     */
+    interface FixedFooter {
+      setPrimaryButton(button: TextButton): FixedFooter;
+      setSecondaryButton(button: TextButton): FixedFooter;
+    }
+
+    /**
+     * Represents a response that makes changes to the calendar event that the user is currently editing in reaction to an action taken in the UI, such as a button click.
+     */
+    interface CalendarEventActionResponse {
+      printJson(): string;
+    }
+
+    /**
+     * A builder for CalendarEventActionResponse objects.
+     */
+    interface CalendarEventActionResponseBuilder {
+      addAttendees(emails: string[]): CalendarEventActionResponseBuilder;
+      build(): CalendarEventActionResponse;
+      setConferenceData(conferenceData: ConferenceData): CalendarEventActionResponseBuilder;
```

Comment:
should be `setConferenceData(conferenceData: Conference_Data.ConferenceData): CalendarEventActionResponseBuilder;`